### PR TITLE
Update privacy area styling

### PR DESCRIPTION
**Pull request by dani-moreno**
## Description

Makes the blur of a privacy area [a bit more contained](https://factorialteam.slack.com/archives/C07LQSJ5UQY/p1732876277064169?thread_ts=1732870097.350609&cid=C07LQSJ5UQY) in its area itself.

## Screenshots

https://github.com/user-attachments/assets/08c97230-a7fb-4022-a05d-85ae3ce9b3ee

---

## Type of Change

- [ ] New experimental component
- [ ] Promote component from experimental to stable
- [x] Maintenance / Bug Fix / Other

### CODE REVIEW

**Review by sauldom102 — CHANGES_REQUESTED** [review]
This is not as straightforward, we are using this to hide chart y axis too

**Comment by dani-moreno** [comment]
> This is not as straightforward, we are using this to hide chart y axis too

Ah, damn, thanks for the heads-up! Is there a story where I could test it, or any way to test it in Storybook?

**Review by sauldom102 — COMMENTED** [review]
I mean this:

Screenshot 2024-11-29 at 15 37 04

File: lib/experimental/Utilities/PrivateBox/index.tsx [hunk 1]
```diff
@@ -1,16 +1,30 @@
...
+          "select-none overflow-hidden rounded-sm ring-f1-border-secondary"
+      )}
       aria-hidden={enabled}
     >
-      {children}
+      <motion.div
+        className="h-full w-full"
+        animate={{
+          opacity: enabled ? 0.3 : 1,
+          filter: enabled ? "blur(5px)" : "blur(0px)",
```
Comment (sauldom102) [inline lib/experimental/Utilities/PrivateBox/index.tsx:22]: We should increase this a little bit more, in the screenshot that I attached you could see the number

Comment (dani-moreno) [reply lib/experimental/Utilities/PrivateBox/index.tsx]: Went with a different approach as we discussed, not using blur at all, which is prone to errors, and just hiding the text instead.

**Review by sauldom102 — APPROVED** [review]

**Pull request merged**